rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/model/EjbSessionBeanModel.java

```diff
@@ -80,11 +80,11 @@
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public JNDIResourceModel getJndiReference();
+    public Iterable<JNDIResourceModel> getJndiReferences();
 
     /**
      * Contains the jndi location for this resource.
      */
     @Adjacency(label = JNDIResourceModel.TYPE, direction = Direction.OUT)
-    public void setJndiReference(JNDIResourceModel jndiReference);
+    public void addJndiReference(JNDIResourceModel jndiReference);
```

Comment:
@bradsdavis but there is another usage that it may bring. In case we would need to access all the global JNDIs, we would simply query the graph for all EJBJNDIs with enum set to GLOBAL.